.github/workflows/build-test-package.yml

```diff
@@ -8,5 +8,7 @@ jobs:
 
   python-build-workflow:
     uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@03626a23c22246e89e36c7e918a158c440f9b099
+    with:
+      manylinux-platforms: '["_2_28-x64","2014-x64"]'
```

Comment:
Thanks very much Tom!

I see at [build-test-package-python.yml](https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/blob/03626a23c22246e89e36c7e918a158c440f9b099/.github/workflows/build-test-package-python.yml#L37 ), the default is `["_2_28-x64","2014-x64","_2_28-aarch64"]`, so you removed `"_2_28-aarch64"` from the default.  👍

Reply:
Yes! I've updated the PR description to make this clearer, thanks for the note.